appcelerator-archive/amp
# Verification token is only usable once

Fixes #1088 

## How to test
```
ampmake build
alias amp='amp -s localhost'
amp cluster create --tag=local
amp user signup --name user --email your@address.email --password password
[Wait for verification email]
$ amp user verify
[localhost:50101]
Your account has now been activated.
$ amp user verify
[localhost:50101]
Error: token has already been used
```
